Makefile

```diff
@@ -3,24 +3,28 @@ all: develop
 
 PIP := python -m pip --disable-pip-version-check
 WEBPACK := yarn build-acceptance
-POSTGRES_CONTAINER := sentry_postgres
 
 freeze-requirements:
 	@python3 -S -m tools.freeze_requirements
 
 bootstrap \
 develop \
+install-js-dev \
+install-py-dev \
+apply-migrations: devenv-sync ;
+
+# This is to ensure devenv sync's only called once if the above
+# aliases are combined e.g. `make install-js-dev install-py-dev`
+.PHONY: devenv-sync
+devenv-sync:
+	devenv sync
+
 clean \
-init-config \
-run-dependent-services \
 drop-db \
 create-db \
-apply-migrations \
 reset-db \
 setup-git \
-node-version-check \
-install-js-dev \
-install-py-dev :
```

Comment:
For now, is it possible to keep the make scripts around while we move over to `devenv`, or allow a `devenv` passthrough to these scripts, e.g. `devenv --pydev`?